lib/gettext-setup/gettext_setup.rb

```diff
@@ -32,8 +33,7 @@ def self.initialize(locales_path, options = {})
 
     # Likewise, be explicit in our default language choice.
     FastGettext.default_locale = default_locale
-    FastGettext.default_available_locales = locales
-
+    FastGettext.default_available_locales = FastGettext.default_available_locales | locales
```

Comment:
Kind of..

This is with ruby 1.9.3

```
irb(main):058:0* x = [1,3,5,6]
=> [1, 3, 5, 6]
irb(main):059:0> y = [2,4,5,7]
=> [2, 4, 5, 7]
irb(main):060:0> x | y
=> [1, 3, 5, 6, 2, 4, 7]
irb(main):061:0> y | x
=> [2, 4, 5, 7, 1, 3, 6]
```
The resultant elements are the same be in a different order